gradle-plugin/openshift/src/main/java/org/eclipse/jkube/gradle/plugin/OpenShiftPlugin.java

```diff
@@ -40,13 +41,15 @@ public OpenShiftPlugin() {
   public Map<String, Collection<Class<? extends Task>>> getTaskPrecedence() {
     final Map<String, Collection<Class<? extends Task>>> ret = new HashMap<>();
     ret.put("ocApply", Collections.singletonList(KubernetesResourceTask.class));
+    ret.put("ocPush", Collections.singletonList(KubernetesBuildTask.class));
```

Comment:
We need to change this to reference the OpenShift tasks too
(Arrays.asList(KubernetesResourceTask.class, OpenShiftResourceTask.class)